src/aktie/index/Index.java

```diff
@@ -779,6 +779,43 @@ public CObjList getMemberships ( String comid, Sort s )
 
         return search ( builder.build(), Integer.MAX_VALUE, s );
     }
+    
+	/**
+	 * Queries all members of a community. Retrieves member identities as
+	 * {@code CObj} from {@code Index}.
+	 * 
+	 * @param communityID
+	 *            The ID of the community.
+	 * @param sort
+	 *            Lucene {@code Sort} specifying the sort order of the result
+	 *            list or {@code null} if no sorting is desired.
+	 * @return A {@code CObjList} referencing the queried members.
+	 */
+	public CObjList getMembers(String communityID, Sort sort) {
+		CObjList memberships = getMemberships(communityID, null);
+
+		BooleanQuery.Builder builder = new BooleanQuery.Builder();
+
+		Term typeTerm = new Term(CObj.PARAM_TYPE, CObj.IDENTITY);
+		builder.add(new TermQuery(typeTerm), BooleanClause.Occur.MUST);
+
+		BooleanQuery.Builder idQuery = new BooleanQuery.Builder();
+
+		for (int i = 0; i < memberships.size(); i++) {
+			CObj membership;
+			try {
+				membership = memberships.get(i);
+			} catch (IOException e) {
+				continue;
```

Comment:
e.printStackTrace()   If we were to ever get an exception here, we'd want the stack trace to understand why.